src/Backend/Synapse/ToStage/StageTableDefinitionFactory.php

```diff
@@ -36,7 +43,8 @@ public static function createStagingTableDefinition(
                             $definition->getColumnDefinition()->getType(),
                             [
                                 'length' => $definition->getColumnDefinition()->getLength(),
-                                'nullable' => true, // set all columns to be nullable
+                                // set all columns to be nullable except in clustered index
+                                'nullable' => !in_array($columnName, $clusteredIndexColumns, true),
```

Comment:
```suggestion
                                'nullable' => $clusteredIndexColumns && !in_array($columnName, $clusteredIndexColumns, true),
```

Reply:
jde jen o to, at se zbytecne ta fce nevola, kdyz to ani nema kde hledat. Vyhodnoceni operatoru je vyrazne levnejsi nez volani fce

Reply:
to same samozrejme na lajne 82 a 105